src/main/java/uk/gov/hmcts/reform/civil/helpers/sdo/SdoHelper.java

```diff
@@ -165,13 +165,12 @@ public static FastTrack getFastTrackAdditionalDirectionEnum(String additionalDir
 
     public static boolean hasFastAdditionalDirections(CaseData caseData, String additionalDirection) {
         FastTrack additionalDirectionEnum = getFastTrackAdditionalDirectionEnum(additionalDirection);
-        List<FastTrack> fastClaims = caseData.getFastClaims();
+        List<FastTrack> fastClaims = caseData.getTrialAdditionalDirectionsForFastTrack() != null
```

Comment:
is caseData.fastClaims included in trialAdditionalDirectionsForFastTrack or is it to be ignored?

Reply:
I'm not sure what you mean. To fully understand this code, see also changes to ccd definition in https://github.com/hmcts/civil-ccd-definition/pull/2726/files